src/etc/inc/radius.inc

```diff
@@ -696,6 +696,26 @@ class Auth_RADIUS extends PEAR {
                     }
                 }
 
+                elseif ($vendor == 13644) { /* Netgate */
+                    switch ($attrv) {
+                    case 1: /* pfSense-Max-Input-Octets */
+                        $this->attributes['maxbytesup'] = radius_cvt_int($datav);
+                        break;
+                    case 2: /* pfSense-Max-Output-Octets */
+                        $this->attributes['maxbytedown'] = radius_cvt_int($datav);
```

Comment:
that's a typo, thank you for spotting it. I fixed it locally, it will be in the next version of the series with any other change required.